### DISCUSSION TITLE
How to get goal custom properties via the Stats API

### DISCUSSION THREAD

**Discussion by waldhacker-joerg** [plausible/analytics#1541] [Self-Hosted Support]
Hi, I'm trying to get the available Custom Properties of the Goals via the Stats API. Unfortunately, I have not found a way. Does anyone know a possibility?

<img width="1202" alt="custom-property-breakdown" src="https://user-images.githubusercontent.com/92523446/146267670-f3e6f300-5bbd-4379-8e14-fa748f79d0c1.png">

**Comment by ukutaht** [comment]
Unfortunately there is no endpoint for that at the moment. It would make a lot of sense to add it!

**Reply by waldhacker1** [reply]
~~I have created a pull request for this: https://github.com/plausible/analytics/pull/1550~~

**Reply by waldhacker1** [reply]
I have created another pull request for this: https://github.com/plausible/analytics/pull/1578
